### PR TITLE
Feature/#149 create menu json tools

### PR DESCRIPTION
**Pull request by manudous**
@brauliodiez create tools

### CODE REVIEW

File: menu-public-portal/src/common/cacheBuster/cacheBuster.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const cacheBuster = (uri: string) => {
```
Comment (brauliodiez) [inline menu-public-portal/src/common/cacheBuster/cacheBuster.ts:1]: Rename folder to 

src/common/tools

Comment (brauliodiez) [reply menu-public-portal/src/common/cacheBuster/cacheBuster.ts:1]: Do not create a folder just only for cacheBuster, this is not going to grow

File: tools/src/pods/restaurant/restaurant.mappers.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import * as model from 'dals';
+import * as apiModel from './restaurant.api-model';
+import { mapToCollection } from 'common/mappers';
+
+export const reduceCategoryEntryListToRationDefinitionList = (
```
Comment (brauliodiez) [inline tools/src/pods/restaurant/restaurant.mappers.ts:5]: Add a comment indicating why all this complexity, my guess is extract from the entries categories that are not there, but add an explanatory commment

File: tools/src/pods/restaurant/restaurant.mappers.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  if (Array.isArray(category)) {
+    const rations = category
+      .map((category: apiModel.CategoryEntry) => category.items)
+      .reduce(
+        (acc: apiModel.Item[], item: apiModel.Item[]) => [...acc, ...item],
+        []
+      )
+      .filter((item: apiModel.Item) => item.priceByRation)
+      .reduce((acc, item: apiModel.Item) => {
+        if (!acc[item.priceByRation.rationName]) {
```
Comment (brauliodiez) [inline tools/src/pods/restaurant/restaurant.mappers.ts:17]: Encapsulate all this inline function in a subfunction and provide a propername

File: tools/src/pods/restaurant/restaurant.mappers.ts [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                ),
+              ]),
+            ],
+          };
+        }
+      }, {});
+
+    return Object.keys(rations).reduce((acc, key) => {
+      return [...acc, { name: key, units: rations[key] }];
+    }, []);
```
Comment (brauliodiez) [inline tools/src/pods/restaurant/restaurant.mappers.ts:42]: Extract this to a function and indicate a proper name indicating what is doing

File: menu-public-portal/src/common/cacheBuster/cacheBuster.ts (hunk 1 above)
Comment (manudous) [reply menu-public-portal/src/common/cacheBuster/cacheBuster.ts]: done

File: tools/src/pods/restaurant/restaurant.mappers.ts (hunk 2 above)
Comment (manudous) [reply tools/src/pods/restaurant/restaurant.mappers.ts:5]: done

File: tools/src/pods/restaurant/restaurant.mappers.ts (hunk 3 above)
Comment (manudous) [reply tools/src/pods/restaurant/restaurant.mappers.ts:17]: done

File: tools/src/pods/restaurant/restaurant.mappers.ts (hunk 4 above)
Comment (manudous) [reply tools/src/pods/restaurant/restaurant.mappers.ts]: done

**Pull request merged**